src/main/java/org/mule/service/http/impl/service/client/CursorNonBlockingInputStreamFeeder.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.service.http.impl.service.client;
+
+import java.io.IOException;
+
+import org.mule.runtime.api.streaming.bytes.CursorStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.ning.http.client.providers.grizzly.FeedableBodyGenerator;
+import com.ning.http.client.providers.grizzly.NonBlockingInputStreamFeeder;
+
+/**
+ * Inputstream feeder used so that we guarantee that a cursor inputstream is properly reset.
+ * 
+ * @since 4.3.0
```

Comment:
Should we specify runtime version or service version?

Reply:
Service version. It's correct your comment.